docker/docker-compose.yml

```diff
@@ -142,50 +113,3 @@ services:
       - "./files/mount:/RackHD/on-tftp/static/tftp"
       - "./monorail:/opt/monorail"
 
-  wss: # 9100
-    build: "../on-wss"
-    depends_on:
-      - files
-      - http
-      - logstash
-      - mongo
-      - rabbitmq
-      - statsd
-      - syslog
-    image: rackhd/on-wss:${TAG}
-    network_mode: "host"
-    privileged: true
-    volumes:
-      - "./monorail:/opt/monorail"
```

Comment:
on-wss is supposed to be used by on-web-ui inside on-http, and suggest not to remove here.